src/resolve.ts

```diff
@@ -316,6 +322,12 @@ function _join(a: string, b: string): string {
   return (a.endsWith("/") ? a : a + "/") + (b.startsWith("/") ? b.slice(1) : b);
 }
 
+function _normalizeWinPath(path: string): string {
+  return path
+    .replace(/\\/g, "/")
+    .replace(/^[A-Za-z]:\//, (r) => r.toUpperCase());
```

Comment:
when would this occur?

even if it does, we can simplify:

```suggestion
    .replace(/^[a-z]:\//, (r) => r.toUpperCase());
```

Reply:
it is same behavior of pathe for path normalization. Anything wrong with driver letter normalization?